public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -313,8 +313,16 @@ function serviceMapQuery(request: DataQueryRequest<TempoQuery>, datasourceUid: s
       const { nodes, edges } = mapPromMetricsToServiceMap(responses, request.range);
       nodes.fields[0].config = {
         links: [
-          makePromLink('Total requests', totalsMetric, datasourceUid),
-          makePromLink('Failed requests', failedMetric, datasourceUid),
+          makePromLink(
+            'Request rate',
+            `rate(${totalsMetric}{server="\${__data.fields.title}"}[$__interval])`,
```

Comment:
Would it be possible to add an additional query with for the client label

Reply:
From what I tested, I don't think there's a great way to include both. I think showing the rate of requests where this node is the server makes the most sense.